src/app/[locale]/(components)/productCard.tsx

```diff
@@ -48,6 +48,7 @@ export const ProductCard: FC<ProductCardProps> = ({
 
   return (
     <Card
+    aria-selected="false"
```

Comment:
can the user still tab through the product cards with this change?

Reply:
They cant tab on them currently. Thats why its a first step :)

Reply:
I have been checking a lot of functionality on many webpages, in general they are pretty bad. I got some good ideas from a bank website. I want I will work to achieve something similar